servers/rendering/rasterizer_rd/rasterizer_canvas_rd.cpp

```diff
@@ -2497,7 +2497,7 @@ RasterizerCanvasRD::~RasterizerCanvasRD() {
 		_dispose_bindings();
 		//anything remains?
 		if (bindings.texture_bindings.size()) {
-			ERR_PRINT("Some texture bindings were not properly freed (leaked canvasitems?");
+			ERR_PRINT("Some texture bindings were not properly freed (leaked canvasitems?)");
```

Comment:
"canvasitems" should be "CanvasItems" (or maybe it's "canvas items" with a typo?).